climpred/checks.py

```diff
@@ -143,3 +143,21 @@ def match_initialized_vars(init, ref):
             f'got {init_vars} for init and {ref_vars} for ref.'
         )
     return True
+
+
+def has_valid_lead_units(xobj):
+    valid_lead_units = ['years', 'seasons', 'months', 'weeks', 'pentads', 'days']
```

Comment:
This should probably move to `constants.py` and make `valid_lead_units` `VALID_LEAD_UNITS` to follow PEP constants convention.